index.html

```diff
@@ -188,7 +189,8 @@ <h2>CANOPY</h2>
         <li><button class="custom-button" type="button">css</button></li>
         <li><button class="custom-button" type="button">JavaScript</button></li>
       </ul>
-      <button id="buttonds5" onclick="popupWindow.classList.add('popup-active')" class="big-button" type="button">See Project</button>
+      <button id="buttonds5" onclick="popupWindow.classList.add('popup-active')" class="big-button" type="button">See
+        Project</button>
     </div>
```

Comment:
- [ ] <b>REQUIRED:</b> The projects section should be created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). This requires changes in existing code - already created HTML code for projects list should be removed and instead it should be added via JavaScript. Check an exampe of students projects: https://github.com/olawale-o/portfolio-mobile-version/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R95 and https://github.com/vladanvi99/my-web-portfolio/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L96.